apollo-router/src/services/supergraph/service.rs

```diff
@@ -125,27 +121,22 @@ impl Service<SupergraphRequest> for SupergraphService {
         let schema = self.schema.clone();
 
         let context_cloned = req.context.clone();
-        let fut = service_call(
-            planning,
-            self.execution_service.clone(),
-            schema,
-            req,
-            self.notify.clone(),
-        )
-        .or_else(|error: BoxError| async move {
-            let errors = vec![
-                crate::error::Error::builder()
-                    .message(error.to_string())
-                    .extension_code("INTERNAL_SERVER_ERROR")
-                    .build(),
-            ];
-
-            Ok(SupergraphResponse::infallible_builder()
-                .errors(errors)
-                .status_code(StatusCode::INTERNAL_SERVER_ERROR)
-                .context(context_cloned)
-                .build())
-        });
+        let fut = service_call(planning, self.execution_service.clone(), schema, req).or_else(
+            |error: BoxError| async move {
+                let errors = vec![
+                    crate::error::Error::builder()
+                        .message(error.to_string())
+                        .extension_code("INTERNAL_SERVER_ERROR")
+                        .build(),
+                ];
+
+                Ok(SupergraphResponse::infallible_builder()
+                    .errors(errors)
+                    .status_code(StatusCode::INTERNAL_SERVER_ERROR)
+                    .context(context_cloned)
+                    .build())
+            },
+        );
```

Comment:
Blame rustfmt for this, the only actual change is that an argument was removed from `service_call`